executor/runtime/docker/docker_linux.go

```diff
@@ -84,6 +90,50 @@ func setupScheduler(cred ucred) error {
 	return nil
 }
 
+func IsCgroup2HybridMode() bool {
+	isHybridOnce.Do(func() {
```

Comment:
Why do we need this sync.Once complexity?

Reply:
this is done to check if the system is in hybrid cgroup mode only once. We dont want to repeat the checks for every container launch.

Reply:
titus-executor is launched once per task. This function will only ever get called one time at start time of this task. setupPostStartLogDirTiniHandleConnection2 only ever gets called once per task. I think this complexity is not needed.

A task may have multiple containers launched though, they will share some of their cgroups though?

Reply:
right, I forgot its a cli. Sure then we dont need this complex logic. Its called when moving pid 1, so only once.